src/components/SearchPane/_search-pane.scss

```diff
@@ -166,7 +166,7 @@ $transition-duration: 0.4s;
 
 .search-pane__overlay {
   @include mq(md) {
-    background: rgba(var(--colour-white), 0.9);
+    background: rgba(255, 255, 255, 0.9);
```

Comment:
Leaving as static value for time being until method of specifying colours values is agreed